_data/sites.json

```
@@ -9135,6 +9153,17 @@
         ]
     },
 
+    {
+        "name": "Surfshark",
+        "url": "https://surfshark.com/privacy-policy",
+        "difficulty": "hard",
+        "notes": "Send a mail to them requesting for account deletion",
```

```suggestion
        "notes": "Send a mail to them requesting for account deletion.",
```

Small formatting nitpick